magentov2/plugin/Indodana/PayLater/Gateway/Request/CaptureRequest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Indodana\Paylater\Gateway\Request;
+
+use Magento\Payment\Gateway\ConfigInterface;
+use Magento\Payment\Gateway\Data\PaymentDataObjectInterface;
+use Magento\Payment\Gateway\Request\BuilderInterface;
+use Magento\Sales\Api\Data\OrderPaymentInterface;
+
+
+class CaptureRequest implements BuilderInterface
```

Comment:
Can you explain what this class for?

Add it on code comment

Reply:
Process of converting an authorized order amount into a billable transaction. Transactions cannot be captured until authorized, and authorizations cannot be captured until the goods or services have been shipped.

![image](https://user-images.githubusercontent.com/32725392/93152401-e15de100-f728-11ea-92ff-6c5d6577a1c6.png)

![image](https://user-images.githubusercontent.com/32725392/93152434-f89cce80-f728-11ea-90e9-dd0e55c816b4.png)

https://devdocs.magento.com/guides/v2.4/payments-integrations/payment-gateway/payment-gateway-intro.html#terms

Reply:
it's to handle capture operations 


Magento payment provider supports the following payment operations:

authorize: process authorization transaction; funds are blocked on customer account, but not withdrawn
sale: process authorization transaction and capture automatically, funds are withdrawn
capture: withdraw previously authorized amount
refund: return previously withdrawn customer funds
void: cancel transfer of funds from customer account

https://devdocs.magento.com/guides/v2.4/payments-integrations/payment-gateway/payment-gateway-intro.html#terms